src/cli/options.ts

```diff
@@ -275,10 +283,16 @@ export const optionMap = {
     alias: '--logo <logo>',
     description: 'The ANT logo',
   },
+  token: {
+    alias: '-t, --token <type>',
+    description: 'Crypto token type for wallet or action',
+    default: 'arweave',
+  },
 };
 
 export const walletOptions = [
   optionMap.walletFile,
+  optionMap.token,
```

Comment:
might not need this when depending on `Signer.signatureType`

Reply:
still necessary to determine which signer class to use, unless we get a clever by dynamically reading the wallet structure